README.md

```diff
@@ -163,280 +173,41 @@ const order: MarketOrderSpec = {
-
-You can avoid using the registry and upload a GVMI image directly to the provider. This is useful when you want to
-quickly prototype your image without having to update the registry with every change.
-
-```ts
-const order: MarketOrderSpec = {
-  demand: {
-    workload: {
-      imageUrl: "file:///path/to/your/image.gvmi",
-    },
-    // other options
-  },
-};
-```
-
-[Check the full example](./examples/advanced//local-image/)
-
-### Setup and teardown methods
+### Read more
 
-You can define a setup method that will be executed the first time a provider is rented and a teardown method
-that will be executed before the rental is done. This is useful when you want to avoid doing the same work
-multiple times when running multiple tasks on the same provider.
+- Read about [other available usage patterns](./docs/USAGE.md) to learn more on how you can leverage the SDK.
+- Check the [`examples` directory](./examples) for working examples showcasing various features of the SDK. All examples
+  are continuously E2E tested.
 
-```ts
-// I want to upload a big file to each provider only once
-const setup: LifecycleFunction = async (exe) => exe.uploadFile("./big-file.txt", "/golem/work/big-file.txt");
-
-// I want to remove the file after I'm done
-const teardown: LifecycleFunction = async (exe) => exe.run("rm /golem/work/big-file.txt");
-
-const pool = await glm.manyOf({
-  order,
-  concurrency,
-  setup,
-  teardown,
-});
-```
-
-[Check the full example](./examples/advanced/setup-and-teardown.ts)
-
-<!--
-TODO:
- ### Market scan
-
-You can scan the market for available providers and their offers. This is useful when you want to see what's available
-before placing an order.
-
-```ts
-await glm.market.scan(order).subscribe({
-  next: (proposal) => {
-    console.log("Received proposal from provider", proposal.provider.name);
-  },
-  complete: () => {
-    console.log("Market scan completed");
-  },
-});
-```
-
-[Check the full example](./examples/basic/market-scan.ts) -->
-
-## Going further
-
-If you wish to learn more about how the SDK functions under the hood, please check out our more advanced examples:
-
-- [Creating pools manually](./examples/advanced/manual-pools.ts)
-- [Performing all market operations manually](./examples/advanced/step-by-step.ts)
-- [(for library authors) Override internal module](./examples/advanced/override-module.ts)
-
-## More examples
+## Examples
 
 The [examples directory](./examples) in the repository contains various usage patterns for the SDK. You can browse
 through them and learn about the recommended practices. All examples are automatically tested during our release
 process.
 
-In case you find an issue with the examples, feel free to submit
-an [issue report](https://github.com/golemfactory/golem-js/issues) to the repository.
-
 You can find even more examples and tutorials in
 the [JavaScript API section of the Golem Network Docs](https://docs.golem.network/docs/creators/javascript).
 
+## Documentation
```

Comment:
IMO link to the docs should be at the very top, even after installation

Reply:
I've decided to keep it here for now.